frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-row.tsx

```diff
@@ -13,17 +14,20 @@ import { vmWizardStorageModalEnhanced } from './vm-wizard-storage-modal-enhanced
 const menuActionEdit = (
   storageWithWrappers: VMWizardStorageWithWrappers,
   { wizardReduxID, withProgress }: VMWizardStorageRowActionOpts,
-): KebabOption => ({
-  label: 'Edit',
-  callback: () =>
-    withProgress(
-      vmWizardStorageModalEnhanced({
-        blocking: true,
-        wizardReduxID,
-        storage: storageWithWrappers,
-      }).result,
-    ),
-});
+): KebabOption => {
+  return {
+    label: 'Edit',
+    isDisabled: storageWithWrappers.volume.name === WINTOOLS_DISK_NAME,
```

Comment:
can we check the type here instead?

Reply:
no, because it can come from `.TEMPLATE` aswell, which we also don't want to edit.

Reply:
ah you are right.
In that case we should also introduce another type `WINDOWS_GUEST_TOOLS_TEMPLATE`  and set it inside the prefillVmTemplateUpdater and check for both of these types.

We should also enhance the deletion of storages of this type on template change to ensure this storage gets always deleted.

Reply:
fixed.
since `WINTOOLS_DISK_NAME` is a constant, I think its cleaner than too have 2 separate types

Reply:
I think it is better to just use it in the least places necessary. - at the template detection side

Any future changes like name change can be done much easier in the future. 
The types are for internal use only and will not be influenced by user input.